### DISCUSSION TITLE
request.validateUsing does not exist

### DISCUSSION THREAD

**Discussion by DJlollipop** [adonisjs/core#4641] [Help]
hello, I would like to use ```request.validateUsing()``` but ```request``` does not have ``validateUsing``` as an argument. I don't know if there is a config to set up or an other thing
thanks you for your help !!

**Comment by RomainLanz** [comment]
Hey @DJlollipop! 👋🏻 

Have you set up `vinejs` for your project?

```sh
node ace add vinejs
```

📚 https://docs.adonisjs.com/guides/basics/validation#configuring-vinejs

**Comment by DJlollipop** [comment]
yes I set up vinejs but I think the problem comes from the types because when I run my code it works. but my IDE subline wherever I use payload

```
async register({ request , response, i18n }: HttpContext) {
    try {
      const payload = await request.validateUsing(registerValidator)
      const user = await User.create({
        email : payload.email,
        name : payload.name,
      })
      const userAuth = await UserAuth.create({
        userId : user.id,
        authType : "credential",
        password : payload.password
      })
      await user.related('auth').save(userAuth)
      response.status(201).send({message : "user has been created"})
    } catch (err) {
      console.log(err)
      response.status(500).send(i18n.t("errors.occurred"))
    }
  }
```
